xml/System.Runtime.InteropServices/SequenceMarshal.xml

```diff
@@ -130,14 +130,14 @@
         <Parameter Name="endIndex" Type="System.Int32" RefType="out" />
       </Parameters>
       <Docs>
-        <typeparam name="T">To be added.</typeparam>
-        <param name="sequence">To be added.</param>
-        <param name="startSegment">To be added.</param>
-        <param name="startIndex">To be added.</param>
-        <param name="endSegment">To be added.</param>
-        <param name="endIndex">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="T">The type of the read-only sequence.</typeparam>
+        <param name="sequence">TThe read-only sequence from which the read-only sequence segment will be retrieved.</param>
+        <param name="startSegment">The beginning read-only sequence segment.</param>
+        <param name="startIndex">The initial position.</param>
+        <param name="endSegment">The ending read-only sequence segment.</param>
+        <param name="endIndex">The final position.</param>
+        <summary>Gets a read-only sequence segment from the underlying read-only sequence.</summary>
+        <returns><see langword="true" /> if it could be retrieved, <see langword="false" /> otherwise.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if the read-only sequence segment can be retrieved; otherwise, <see langword="false" />.</returns>
```